client/flutter/lib/pages/home_page.dart

```diff
@@ -168,23 +168,31 @@ class _HomePageState extends State<HomePage> {
                         Icon(Icons.arrow_forward_ios)
                       ],
                     ),
-                    onPressed: () =>
-                        launch("https://www.who.int/Covid19ResponseFund")),
+                    onPressed: () {
+                        _logAnalyticsEvent('Donate');
+                        launch("https://www.who.int/Covid19ResponseFund");
+                    })
               ),
               ListTile(
                 leading: Icon(Icons.share),
                 title: Text(S.of(context).homePagePageSliverListShareTheApp),
                 trailing: Icon(Icons.arrow_forward_ios),
-                onTap: () => Share.share(
-                    S.of(context).commonWhoAppShareIconButtonDescription),
+                onTap: () {
+                  _logAnalyticsEvent('Share');
```

Comment:
Use the standard logShare() method instead of regular logEvent().
The share event with be helpful in the analytics dashboard.
https://pub.dev/documentation/firebase_analytics/latest/firebase_analytics/FirebaseAnalytics/logShare.html

Reply:
Yes, I think I accidentally removed your initial logShare() during a merge. Thanks for catching this!